packages/venia-concept/src/components/Header/header.js

```diff
@@ -1,79 +1,77 @@
-import React, { Component, Suspense } from 'react';
-import PropTypes from 'prop-types';
+import React, { Suspense } from 'react';
+import { bool, func, shape, string } from 'prop-types';
+import MenuIcon from 'react-feather/dist/icons/menu';
+import SearchIcon from 'react-feather/dist/icons/search';
 
-import classify from 'src/classify';
-import { Link, resourceUrl, Route } from 'src/drivers';
 import Icon from 'src/components/Icon';
-import SearchIcon from 'react-feather/dist/icons/search';
-import MenuIcon from 'react-feather/dist/icons/menu';
+import Logo from 'src/components/Logo';
+import { Link, resourceUrl, Route } from 'src/drivers';
+
 import CartTrigger from './cartTrigger';
 import NavTrigger from './navTrigger';
 import SearchTrigger from './searchTrigger';
 
-const SearchBar = React.lazy(() => import('src/components/SearchBar'));
-
+import { mergeClasses } from 'src/classify';
 import defaultClasses from './header.css';
-import Logo from '../Logo';
-
-class Header extends Component {
-    static propTypes = {
-        classes: PropTypes.shape({
-            logo: PropTypes.string,
-            primaryActions: PropTypes.string,
-            root: PropTypes.string,
-            open: PropTypes.string,
-            closed: PropTypes.string,
-            secondaryActions: PropTypes.string,
-            toolbar: PropTypes.string
-        }),
-        searchOpen: PropTypes.bool,
-        toggleSearch: PropTypes.func.isRequired
-    };
 
-    get searchIcon() {
-        return <Icon src={SearchIcon} />;
-    }
+const SearchBar = React.lazy(() => import('src/components/SearchBar'));
```

Comment:
I'm new to `React.lazy`. How can I see that this is indeed lazily loading? What affect should it have in the browser? I'm not seeing a new chunk/request being made which is confusing me :D